### PR TITLE
Dataman toplevelscreens

### PR DESCRIPTION
**Pull request by cr1spyvada**
- [x] Add Bottom Tab Navigation
- [x] Create Home Screen 
- [x] Create Notifications Screen
- [x] Create Accounts Screen
- [x]  Added Native Wind Support

### CODE REVIEW

**Comment by sutyum** [comment]
Add a task checklist to the description of the pr. refer to other merger prs on this repo for reference. @cr1spyvada 
Request for a review when the task is done

**Review by chandantiwari1 — COMMENTED** [review]
no issue

File: apps/Dataman/src/screens/account.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            }}
+          >
+            Mark Attendance
+          </Button>
+        </View>
+      </View>
+    </View>
+  );
+};
+const styles = StyleSheet.create({
```
Comment (sutyum) [inline apps/Dataman/src/screens/account.tsx]: Why is this needed? why can this not be done using tailwind?

File: apps/Dataman/src/screens/home.tsx [hunk 2]
```diff
@@ -1,9 +1,79 @@
...
+              {item}
+            </List.Item>
+          ))}
+        </List>
+      </View>
     </View>
   );
 };
+
+const styles = StyleSheet.create({
```
Comment (sutyum) [inline apps/Dataman/src/screens/home.tsx]: tailwind

File: apps/Dataman/src/screens/home.tsx [hunk 3]
```diff
@@ -1,9 +1,79 @@
 import * as React from "react";
-import { View, Text } from "react-native";
+import { StyleSheet, Text } from "react-native";
+import { Card, Carousel, List, View } from "@ant-design/react-native";
+import { AntDesign } from "@expo/vector-icons";
+
 export const HomeScreen = () => {
+  const listMap = [
```
Comment (sutyum) [inline apps/Dataman/src/screens/home.tsx]: use a better/less generic name

File: apps/Dataman/src/screens/notifications.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            >
+              {item}
+            </List.Item>
+          ))}
+        </List>
+      </View>
+    </View>
+  );
+};
+const styles = StyleSheet.create({
```
Comment (sutyum) [inline apps/Dataman/src/screens/notifications.tsx]: tailwind

**Review by sutyum — CHANGES_REQUESTED** [review]

**Review by sutyum — APPROVED** [review]

**Pull request merged**